source_code/main_mcu/src/GUI/gui_prompts.c

```diff
@@ -1171,24 +1193,39 @@ RET_TYPE gui_prompts_get_user_pin(volatile uint16_t* pin_code, uint16_t stringID
         /* Click + up/down action */
         if ((detection_result == WHEEL_ACTION_CLICK_UP) || (detection_result == WHEEL_ACTION_CLICK_DOWN))
         {
-            // Do nothing
-            detection_result = WHEEL_ACTION_NONE;
+            // If Click + up
+            if (detection_result == WHEEL_ACTION_CLICK_UP)
```

Comment:
you're overriding default behavior here, can you add a if(show_pin_entry)?

Reply:
You're right! Adding the condition.